ani.ter-markosyan/repo/Lesson_6/exercise_11/exercise_11.py

```diff
@@ -0,0 +1,10 @@
+#!usr/bin/evn python3
+
+list = []
+word = str
```

Comment:
LW: What is this line for? And this is also a wrong declaration. Use type() to ensure